src/neuroconv/tools/testing/_mock/_mock_dataset_models.py

```diff
@@ -7,46 +7,48 @@
 from ...nwb_helpers import (
     AVAILABLE_HDF5_COMPRESSION_METHODS,
     AVAILABLE_ZARR_COMPRESSION_METHODS,
-    DatasetInfo,
     HDF5BackendConfiguration,
     HDF5DatasetIOConfiguration,
     ZarrBackendConfiguration,
     ZarrDatasetIOConfiguration,
 )
 
 
-def mock_DatasetInfo(
+def mock_HDF5DatasetIOConfiguration(
```

Comment:
mock at the beginning of the name. It could also be mock_DatasetIOConfigurarion(backend: ["hdf5", "zarr"])) to save some code duplcation. 

But I am not sure is better. This also works given that we have only two objects.